projects/igniteui-angular/migrations/common/util.ts

```diff
@@ -100,15 +100,15 @@ export const tryInstallPackage = (context: SchematicContext, packageManager: str
         context.logger.debug(`Installing ${pkg} via ${packageManager}.`);
         switch (packageManager) {
             case 'yarn':
-                execSync(`${packageManager} add ${pkg} --no-lock-file`, { stdio: 'ignore' });
+                execSync(`${packageManager} add ${pkg}`, { stdio: 'ignore' });
                 break;
             case 'npm':
                 execSync(`${packageManager} i ${pkg} --no-save --no-audit`, { stdio: 'ignore' });
                 break;
         }
         context.logger.debug(`${pkg} installed successfully.`);
     } catch (e) {
-        context.logger.warn(`Could not install ${pkg}.`, JSON.parse(e));
+        context.logger.warn(`Could not install ${pkg}.`, e.stringify());
```

Comment:
Is there a reason why we don't use `JSON.stringify` here but a `stringify` method that's on the error object?